### PR TITLE
Feature create routes

### PR DESCRIPTION
**Pull request by danstewart**

### CODE REVIEW

**Comment by danstewart** [comment]
Had to force push to fix the author email :angel:

**Comment by Williams-Dan** [comment]
Looks good, but could you please put the new routes in a new user controller and put the templates in a users folder under views?

File: app/controllers/user_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require './config/environment'
+
+#:nodoc:
+class UserController < Sinatra::Base
```
Comment (Williams-Dan) [inline app/controllers/user_controller.rb:6]: Inherit ApplicationController  plox :)

Comment (danstewart) [reply app/controllers/user_controller.rb:6]: I can't get that to work without breaking the `ApplicationController`'s layout

Comment (Williams-Dan) [reply app/controllers/user_controller.rb:6]: Meh Ship it!

**Pull request merged**